### PR TITLE
Added pagination to the rest api

### PR DESCRIPTION
**Pull request by terrasea2**
Also cleaned up code and removed some unneeded code

Added docker-compose file and modified Dockerfile so Dockerfile actually builds

Trello Card: https://trello.com/c/BoaYXVa8/3172-5-some-all-journals-are-not-getting-crossref-citations

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    environment:
+      - MYSQL_DATABASE=altm
+      - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+    expose:
+      - 3306
+
+  articlemetrics:
+    build: .
+    environment:
+      - DJANGO_SETTINGS_MODULE=core.dev_settings
```
Comment (rowan08) [inline docker-compose.yml]: I'm not hugely familiar with docker-compose. Can this value be set at runtime or does it need to be changed to `settings_live`/`settings_docker` here?

Comment (terrasea2) [reply docker-compose.yml]: It is set in the Dockerfile. I put it here to override what was in Dockerfile. It can be set by docker-compose

Comment (rowan08) [reply docker-compose.yml]: ok, but is this `docker-compose.yml` used in the live environment?

Comment (terrasea2) [reply docker-compose.yml]: No, not at this time, and I doubt it will ever be.

**Pull request merged**